### DISCUSSION TITLE
Adding Coinbase payment method

### DISCUSSION THREAD

**Discussion by NicolasDeGouveia** [swellstores/community#262] [Help] [answered]
Hi, i'm a front end developer and i'm currently working on testing the swell solution for my job. I successfully setup swell but now i'm facing an "issue". I wanted to add the possibility for users to pay with cryptocurrencie so i created a Rally account and i attach my Swell account to it. But now i really don't know how to manage to get the good checkout page on my project wich propose crypto payment method.

For the moment, it's still the checkout page that i receive from the cart.get method "checkout_url" and it's not that one with the crypto payment method.

I didn't find any documentations, i just followed the steps that are mentionned here : https://www.swell.is/blog/add-crypto-payment-integrations-to-your-ecommerce-store

Thank you for your help

Nicolas

**Comment by zenkkor** [comment, marked as answer]
Hey Nicolas! 👋 

Rok here, responding from the Rally team. It looks like you went through our self-service onboarding, so you likely missed a step in your Rally admin guided onboarding.

I'll send you an email as well, but in short here are the steps to follow to get Rally live on Swell:
1. Finish onboarding in the Rally Admin by following our simple guided onboarding
2. Integrate Rally to your storefront theme by following our [developer docs for Swell](https://developers.rallyon.com/docs/elements/storefront/swell/)
      a. The `rallyClientId` can be found under "Settings" -> "Account info", then scroll down to the "Platform" section and click the three dots, then "Settings". You can copy-paste the "Rally client ID" from the modal.
4. Go online by clicking the "Go Online" button in "Settings" -> "Account info" (first tab under "Storefront").

I'll send you an email, so you have a direct contact in case you get stuck or need any help.

**Reply by NicolasDeGouveia** [reply]
Hey zenkkor,

I'll try this and if i get stuck i will contact you directly.

Thank you for your help

**Reply by zenkkor** [reply]
Sounds like a plan!
